src/averageAges.js

```diff
@@ -55,7 +73,21 @@ function calculateWomenAverageAge(people, withChildren) {
  * @return {number}
  */
 function calculateAverageAgeDiff(people, onlyWithSon) {
-  // write code here
+  const kids = people.filter(kid => {
+    const mothers = people.some(person => kid.mother === person.name);
```

Comment:
better rename to hasMother, it would be more representative of this const's function